Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -86,11 +86,13 @@
 MAILBOX = "mailbox"
 MAILBOX_ID = "mailboxId"
 FOLDER_ID = "id"
+TARGET_MAILBOX = 'receivedBy'
 
 # context paths
-CONTEXT_UPDATE_EWS_ITEM = "EWS.Items(val.{0} === obj.{0} || (val.{1} && obj.{1} && val.{1} === obj.{1}))".format(
-    ITEM_ID, MESSAGE_ID
-)
+CONTEXT_UPDATE_EWS_ITEM = f"EWS.Items((val.{ITEM_ID} === obj.{ITEM_ID} || " \
```

Comment:
why do we use `or` between the item ID and the message ID conditions? can the item ID be missing?

Reply:
not sure why but I didn’t want to remove it.